packages/replay/src/coreHandlers/handleDom.ts

```diff
@@ -47,13 +48,21 @@ function handleDom(handlerData: DomHandlerData): Breadcrumb | null {
     return null;
   }
 
+  const serializedNode = targetNode && ('__sn' in targetNode) ? targetNode.__sn  as elementNode & {id: number} : null;
   return createBreadcrumb({
     category: `ui.${handlerData.name}`,
     message: target,
     data: {
-      // Not sure why this errors, Node should be correct (Argument of type 'Node' is not assignable to parameter of type 'INode')
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ...(targetNode ? { nodeId: record.mirror.getId(targetNode as any) } : {}),
+      ...(serializedNode ? {
+        nodeId: serializedNode.id,
+        node: {
+          id: serializedNode.id,
+          tagName: serializedNode.tagName,
+          textContent: targetNode ? Array.from(targetNode.childNodes).filter((node: Node | INode) => '__sn' in node && node.__sn.type === NodeType.Text).map(node => node.textContent) : '',
```

Comment:
I plan on capping the size of this value to 256 characters.  No action item but I thought I would mention it.

Reply:
@cmanallen should backend add additional metadata when it changes the payload? This means frontend could more easily tell user what payloads were modified and why.